datasets/ms_terms/README.md

```diff
@@ -0,0 +1,124 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```

Comment:
Could you add the tags here please ?
You can find more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

If you have issues when loading the dataset in the tagger, then restart the tagger and create the tags without loading it. For this dataset it should do the job.

Reply:
I gave it another try today. It works (in a sense that there was no error when I clicked Done? Save to File!). I did not find a file created. The "Show YAML output aggregating the tags saved for all configs" button gave a few lines of information which I have put into README.md. Let me know if more need to be done.